skeleton/template.yaml

```diff
@@ -192,15 +192,14 @@ spec:
           image: quay.io/redhat-appstudio/dance-bootstrap-app:latest # bootstrap app image as placeholder
           # actual src image, should be used by tekton PR pipeline to update the image in gitops repo
           srcImage: '${{ parameters.imageRegistry }}/${{ parameters.imageOrg }}/${{ parameters.imageName }}'
-          port: sed.edit.IMAGEPORT
           argoNS: ${ARGOCD__DEFAULT__NAMESPACE}
           argoProject: ${ARGOCD__DEFAULT__PROJECT}
           secretRef: ${{ parameters.hostType === 'GitLab' }}
           gitSecret: ${GIT__SECRET__DEFAULT__NAME}
           gitSecretKey: ${GIT__SECRET__DEFAULT__KEY}
           webhookSecret: ${WEBHOOK__SECRET__DEFAULT__NAME}
           webhookSecretKey: ${WEBHOOK__SECRET__DEFAULT__KEY}
-          defaultBranch: main
+          defaultBranch: ${{ parameters.branch }}
```

Comment:
we probably want to leave gitops repo default branch as `main` to align with rhtap samples.  we only allow user to customize the source repo's default branch & repo name.

Reply:
Makes sense, updated!